File: ids4-samples/netCore/ids4-sample/Entities/ReturnObject.cs
namespace ids4_sample.Entities
{
    public class ReturnObject
    {
        public ReturnObject(
            Token tokenData, DemoApiResult result
        ) {
            Success = true;
            ClientCredentialsToken = tokenData;
            SampleApiResult = result;
        }

        public bool Success { get; set; }
        public Token ClientCredentialsToken { get; set; }
        public DemoApiResult SampleApiResult { get; set; }
    }
}

File: ids4-samples/nestjs/ids4-sample/src/entities/return-object.ts
import { DemoApiResult } from './demo-api-result';
import { Token } from "./token";

export class ReturnObject {
    constructor(tokenData: Token, result: DemoApiResult) {
        this.success = false;
        this.clientCredentialsToken = tokenData;
        this.sampleApiResult = result;
    }

    success: boolean;
    clientCredentialsToken: Token;
    sampleApiResult: DemoApiResult;
}